src/features/QueueConsumer.js

```diff
@@ -43,7 +43,10 @@ var QueueConsumer = function (solaceModule, queueName) {
     };
 
     consumer.log('\n*** Consumer to queue "' + consumer.queueName + '" is ready to connect ***');
-
+    consumer.log('\n/*********************************************************************' +
+    '\nNOTE: Before you proceed, please ensure that the required queue is created on the message router vpn.' +
+    '\n/*********************************************************************')
+  
```

Comment:
We need to add a similar comment in the `Guaranteedsubscriber.js` sample

Reply:
It is already there...double-checked!!

    subscriber.log('\n*** Consumer to queue "' + subscriber.queueName + '" is ready to connect ***');
    subscriber.log('\n/*********************************************************************' +
                    '\nNOTE: Before you proceed, please ensure that the required queue with ' +
                    '\nappropriate subscription is created on the Broker.' +
                    '\n/********************************************************************/')

I just updated the "message router VPN" -. "Broker".